src/mono/wasm/runtime/exports.ts

```diff
@@ -20,15 +20,16 @@ import {
     mono_wasm_raise_debug_event,
     mono_wasm_fire_debugger_agent_message,
 } from "./debug";
-import { runtimeHelpers, setLegacyModules } from "./modules";
-import { EmscriptenModuleMono, MonoArray, MonoConfig, MonoConfigError, MonoObject } from "./types";
+import { runtimeHelpers, setAPI } from "./modules";
+import { EmscriptenModuleConfig, EmscriptenModuleMono, MonoArray, MonoConfig, MonoConfigError, MonoObject } from "./types";
 import {
     mono_load_runtime_and_bcl_args, mono_wasm_load_config,
     mono_wasm_setenv, mono_wasm_set_runtime_options,
     mono_wasm_load_data_archive, mono_wasm_asm_loaded,
     mono_wasm_set_main_args,
     mono_wasm_pre_init,
-    mono_wasm_on_runtime_initialized
+    mono_wasm_on_runtime_initialized,
+    runtime_is_initialized
```

Comment:
why no mono_wasm prefix?

Reply:
Confusingly, there is also `mono_wasm_runtime_ready` and `mono_wasm_runtime_is_ready` for debugger, meaning something very different. 

Long term, I would like to drop `mono_wasm` prefix from internal code and leave it only on API.

Right now I added it for consistency sake.